test/unit/sanitize/filters/unique-id-prefix.js

```diff
@@ -0,0 +1,80 @@
+/*globals describe, it*/
+require('should');
+
+describe('svg-react-loader/lib/sanitize/filters/unique-svg-ids', () => {
```

Comment:
Maybe you can also include a simple test in `test/integration/test.js` on an actual svg file.